packages/react-components/src/components/Datepicker/RangeDatePicker.tsx

```diff
@@ -0,0 +1,248 @@
+import * as React from 'react';
+import {
+  isAfter,
+  isSameDay,
+  subMonths,
+  differenceInCalendarDays,
+} from 'date-fns';
+import {
+  calculateDatePickerMonth,
+  getRangeDatePickerModifiers,
+  getSelectedOption,
+  isDateWithinRange,
+  isSelectingFirstDay,
+} from './helpers';
+
+import { useRangeDatePickerState } from './hooks';
+import {
+  IRangeDatePickerChildrenPayload,
+  IRangeDatePickerOption,
+  IRangeDatePickerProps,
+  RangeDatePickerAction,
+} from './types';
+
+export const RangeDatePicker = (
+  props: IRangeDatePickerProps
```

Comment:
please spread props here
```suggestion
 { .... }: IRangeDatePickerProps
```

Reply:
👌🏼